docs/create_doc_files.sh

```diff
@@ -64,7 +64,9 @@ for f in ../examples/*.py; do
         echo >> $rst
         echo "$pdf" >> $rst
         echo >> $rst
-        echo ".. figure:: /_static/examples/${pdf}.png" >> $rst
+        for figure in ${pdf}*.png; do
```

Comment:
To support test files that create tests with multiple pages (ie. multiple figures).